### ISSUE TITLE
aws.ec2.subnets: OperationNotPageableError

### ISSUE THREAD

**Issue by achantavy** [lyft/cartography#431]
*Description*:
> What issue is being seen?  Describe what should be happening instead of the bug, for example: Cartography should not crash, the expected value isn't returned, the data schema is wrong, etc.

AWS EC2 Subnet sync crashes with botocore.exceptions.OperationNotPageableError, Operation cannot be paginated: describe_subnets.

*To Reproduce*:
> Steps to reproduce the behavior.  Provide all data and inputs required to reproduce the issue.

Run the sync.

*Logs*:
> If applicable, copy and paste your console log with the failing stack trace.

```
  File "{PATH}/cartography/intel/aws/__init__.py", line 98, in _sync_multiple_accounts
    _sync_one_account(neo4j_session, boto3_session, account_id, sync_tag, common_job_parameters)
  File "{PATH}/cartography/intel/aws/__init__.py", line 45, in _sync_one_account
    ec2.sync(neo4j_session, boto3_session, regions, account_id, sync_tag, common_job_parameters)
  File "{PATH}/cartography/util.py", line 58, in timed
    result = method(*args, **kwargs)
  File "{PATH}/cartography/intel/aws/ec2/__init__.py", line 35, in sync
    sync_subnets(neo4j_session, boto3_session, regions, account_id, sync_tag, common_job_parameters)
  File "{PATH}/cartography/util.py", line 58, in timed
    result = method(*args, **kwargs)
  File "{PATH}/cartography/intel/aws/ec2/subnets.py", line 78, in sync_subnets
    data = get_subnet_data(boto3_session, region)
  File "{PATH}/cartography/util.py", line 58, in timed
    result = method(*args, **kwargs)
  File "{PATH}/cartography/util.py", line 78, in inner_function
    return func(*args, **kwargs)
  File "{PATH}/cartography/intel/aws/ec2/subnets.py", line 15, in get_subnet_data
    paginator = client.get_paginator('describe_subnets')
  File "{PATH}/botocore/client.py", line 735, in get_paginator
    raise OperationNotPageableError(operation_name=operation_name)
botocore.exceptions.OperationNotPageableError: Operation cannot be paginated: describe_subnets
```

https://github.com/lyft/cartography/blob/1fa46905e8c96cd82a399d7567f8451a6052f35c/cartography/intel/aws/ec2/subnets.py#L15

*Please complete the following information:*:
>  - Cartography release version or commit hash [e.g. 0.12.0 or 95e8e11913e2a44a4d4682506d8364a638ceac69]

1fa46905e8c96cd82a399d7567f8451a6052f35c

Boto3 version: `boto3==1.9.134`

Anyone else seeing this? cc: @krisek @marco-lancini

**Comment by krisek** [comment]
Hi,

I have on my dev environment

```
(cartography-venv)  cartography-venv ~/p/cartography $  pip freeze | grep boto                                              boto3==1.15.1
botocore==1.18.1
```

and in our prod:

```
(venv-cartography) cartography /cartography (master)$ pip freeze | grep boto
boto3==1.15.2
botocore==1.18.2
```

and never seen. 

Catching the exception doesn't solve the problem... quick research didn't provide any solution, but boto is sometimes broken, so we'll need to take time and look into the details.

**Comment by achantavy** [comment]
Ah thanks for this, I didn't realize my boto version was this far behind. I'll retry with a newer version.

**Comment by achantavy** [comment]
Fixed in #432
